deep-deep/setup.py

```diff
@@ -27,13 +27,14 @@
     ],
     install_requires=[
         'psutil',
-        'scrapy >= 1.1',
-        'scikit-learn >= 0.17.1',
+        'scrapy >= 1.3.2',
+        'w3lib >= 1.17.0',
+        'scikit-learn >= 0.18.1',
         'joblib',
         'numpy',
         'scrapy-cdr',
         'json-lines >= 0.3.1',
-        'html-text >= 0.1.1',
+        'html-text >= 0.2.1',
```

Comment:
I think that's fine - last year's models won't work anyway, and Sitehound models are usually re-trained anyway, so we can do it.